src/views/studio/lib/studio-member-actions.js

```diff
@@ -15,6 +15,7 @@ const Errors = keyMirror({
     UNKNOWN_USERNAME: null,
     RATE_LIMIT: null,
     MANAGER_LIMIT: null,
+    CONFLICT: null,
```

Comment:
I wonder if we should name this something more descriptive of the problem (like CANNOT_BE_HOST)? I chose 409 (ConflictError) as the response because it was close to what we wanted, but it is still sort of a stretch, so it feels like something more descriptive in the frontend would help clarify things.

Reply:
Good point - I'll update it